psir_demo_moveit_config/config/controllers.yaml

```diff
@@ -1,5 +1,5 @@
 controller_list:
-  - name: "/psir/arm_joint_trajectory_controller"
+  - name: "/prbt/manipulator_joint_trajectory_controller"
```

Comment:
shouldn't this be `/prbt/arm_joint_trajectory_controller`?

Reply:
The controller group name has changed